README.md

```diff
@@ -238,6 +238,16 @@ No. This approaches are not equivalent how they work. The problem that
 plugin may not find some of kotlin plugins if both approaches are used
 in the project configuration. Especially it is related to Android plugin.
 
+- How can I import `ReporterType` in my external Gradle script?
+
+Gradle doesn't allow to have import statements in external gradle script files (for example: `quality/foo.gradle`).
```

Comment:
What?! This sounds like a bug in Gradle. Care to open or link to an existing bug here?